client/influxdb.go

```diff
@@ -183,6 +239,52 @@ func (c *Client) Write(bp BatchPoints) (*Response, error) {
 	return nil, nil
 }
 
+// WriteLineProtocol takes a string with line returns to delimit each write
+// If successful, error is nil and Response is nil
+// If an error occurs, Response may contain additional information if populated.
+func (c *Client) WriteLineProtocol(data, database, retentionPolicy, precision, writeConsistency string) (*Response, error) {
+	c.url.Path = "write"
```

Comment:
Should make a copy of `c.url` and modify that so multiple go routines could use the same client to make requests.